scripts/get_dev_tools.sh

```diff
@@ -7,15 +7,4 @@ set -xeuo pipefail
 go get github.com/golang/lint/golint
 go get github.com/golang/dep/cmd/dep
 
-GO_VERSION="go1.11beta3"
```

Comment:
what i liked on this is that this allowed me to use the stable version of Go on my dev env. while new way requires me to stay up to date with latest release

Reply:
@michalpristas you can still have stable/unstable versions in your dev env. This is just a set up script, so I'm not sure how you're using it. In any case, we can potentially separate the CI set up script from the dev set up script if it makes sense

Reply:
I think splitting up the scripts for dev vs CI makes sense. Maybe during #397

Reply:
Ref this thread: https://github.com/gomods/athens/pull/418#discussion_r208064390